salt/salt/hawkdev/init.sls

```diff
@@ -75,3 +75,19 @@ hawk-development:
     - group: root
     - mode: 755
 
+salt://utils/configure_nagios_server.sh:
+  cmd.script:
+    - runas: root
+
+/usr/local/nagios/etc/objects/cluster_api.cfg:
+  file.managed:
+    - source: salt://files/cluster_api.cfg
+    - template: jinja
+
+nagios:
+  service.running: []
```

Comment:
Do you need a dependency on the files here so things happen in the right order?

Like

```
nagios:
  service.running:
    - require:
      - file: /usr/local/nagios/etc/objects/cluster_api.cfg
      - cmd: salt://utils/configure_nagios_server.sh
```

like in https://docs.saltstack.com/en/latest/ref/states/requisites.html

Reply:
Brilliant!! and fixed :)